cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -68,6 +68,7 @@
       cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'cron', 'index_users_in_solr')
       cronjob at:'25 7 * * *', do:deploy_dir('bin', 'cron', 'update_hoc_map')
       cronjob at:'*/10 * * * *', do:deploy_dir('bin', 'cron', 'delete_twilio_data')
+      cronjob at:'18 13-21 * * *', do:deploy_dir('bin', 'cron', 'confirm_usage')
```

Comment:
Note that this is 8 AM to 4 PM Central. This seems to capture "school" hours across the country, though the thresholds are currently such that we should meet them during weekends over the summer.